draft-ietf-quic-transport.md

```diff
@@ -1610,7 +1610,7 @@ A resumption token SHOULD be constructed to be easily distinguishable from
 tokens that are sent in Retry packets as they are carried in the same field.
 
 If the client has a token received in a NEW_TOKEN frame on a previous connection
-to what it believes to be the same server, it can include that value in the
+to what it believes to be the same server, it SHOULD include that value in the
```

Comment:
I think that we need text on why it might not.  I suspect that this is a MUST unless (sending from a new address) OR (you don't feel like it).

I think that the "can" is fine here.

Reply:
... or it dropped its store of tokens. SHOULD is useful in encouraging clients to do the performant thing -- use tokens unless they really can't.

Reply:
If we go with SHOULD, then we need to address the reasons that an endpoint might diverge from that.  I realize that it's a cop-out, but "can" is so much easier.